src/pages/profile/Profile.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Navbar } from "../../components/index";
+import "./profile.css";
+import { NavLink } from "react-router-dom";
+
+function Profile() {
+  const username = localStorage.getItem("username");
+  const email = localStorage.getItem("email");
+
+  return (
+    <div>
+      <Navbar />
+
+      <ul className="categories_navTabs_list mb-2">
+        <li>
+          <NavLink
+            to="/profile"
+            className="categories_navLink categories_navLink1"
+          >
+            Profile
+          </NavLink>
+        </li>
+
+        <li>
+          <NavLink
+            to="/profile"
+            className="categories_navLink categories_navLink1"
+          >
+            Addresses
+          </NavLink>
+        </li>
+
+        <li>
+          <NavLink
+            to="/profile"
+            className="categories_navLink categories_navLink1"
+          >
+            Orders
+          </NavLink>
+        </li>
+
+        <li>
+          <NavLink
+            to="/profile"
+            className="categories_navLink categories_navLink1"
+          >
+            Setting
+          </NavLink>
+        </li>
```

Comment:
suggestion: Since it is a repeating structure you can use map.

Reply:
Yess , right I will change that